modules/raft/src/test/java/org/apache/ignite/raft/jraft/rpc/TestMessageGroup.java

```diff
@@ -6,32 +6,22 @@
  * (the "License"); you may not use this file except in compliance with
  * the License.  You may obtain a copy of the License at
  *
- *     http://www.apache.org/licenses/LICENSE-2.0
+ *      http://www.apache.org/licenses/LICENSE-2.0
  *
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.ignite.raft.jraft.rpc.message;
 
-import org.apache.ignite.raft.jraft.rpc.RpcRequests;
+package org.apache.ignite.raft.jraft.rpc;
 
-class PingRequestImpl implements RpcRequests.PingRequest, RpcRequests.PingRequest.Builder {
-    private long sendTimestamp;
+import org.apache.ignite.network.annotations.MessageGroup;
 
-    @Override public long getSendTimestamp() {
-        return sendTimestamp;
-    }
-
-    @Override public Builder setSendTimestamp(long timestamp) {
-        this.sendTimestamp = timestamp;
-
-        return this;
-    }
-
-    @Override public RpcRequests.PingRequest build() {
-        return this;
-    }
+/**
+ * Message group for tests.
+ */
+@MessageGroup(groupType = 4, groupName = "TestRaftMessages")
```

Comment:
I think we should use more obscure numbers for test groups, what do you think?

Reply:
why?

Reply:
I'm afraid that we'll have collisions

Reply:
where? I can only see a problem if you start using test messages from one module in another, but I don't think this is a viable use case.

Reply:
Ok